.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test/3rdParty/babylon"]
+	path = test/3rdParty/babylon
+	url = https://github.com/babel/babylon.git
```

Comment:
you don't want that to be a regular dep. You could add a dev-dep of `"babel-fixtures":  "babel/babylon#6.13.1"`